### PR TITLE
Add examples for github:repo* scaffolder actions

### PR DESCRIPTION
**Pull request by kurtaking**
#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

## Details

Add examples for `github:repo-create` and `github:repo:push`.

### github:repo:create

image

### github:repo:push

image

### CODE REVIEW

File: plugins/scaffolder-backend/src/scaffolder/actions/builtin/github/githubRepoCreate.examples.test.ts
Comment (kurtaking) [inline plugins/scaffolder-backend/src/scaffolder/actions/builtin/github/githubRepoCreate.examples.test.ts:1]: The test was copied from `githubRepoCreate.test.ts` and then modified.

File: plugins/scaffolder-backend/src/scaffolder/actions/builtin/github/githubRepoPush.examples.test.ts
Comment (kurtaking) [inline plugins/scaffolder-backend/src/scaffolder/actions/builtin/github/githubRepoPush.examples.test.ts:1]: The test was copied from `githubRepoPush.test.ts` and then modified.

File: plugins/scaffolder-backend/src/scaffolder/actions/builtin/github/githubRepoCreate.examples.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      ],
+    }),
+  },
+  {
+    description: 'Add a description.',
+    example: yaml.stringify({
+      steps: [
+        {
+          action: 'github:repo:create',
+          name: 'Create a new GitHub repository',
```
Comment (kurtaking) [inline plugins/scaffolder-backend/src/scaffolder/actions/builtin/github/githubRepoCreate.examples.ts:41]:
```suggestion
          name: 'Create a new GitHub repository with a description',
```

**Comment by kurtaking** [comment]
Related issue: https://github.com/backstage/backstage/issues/18832.

File: .changeset/orange-pandas-worry.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': patch
+---
+
+docs: add examples for github:repo:create and github:repo:push scaffolder actions
```
Comment (Rugvip) [inline .changeset/orange-pandas-worry.md:5]:
```suggestion
Add examples for `github:repo:create` and `github:repo:push` scaffolder actions.
```

**Review by Rugvip — APPROVED** [review]
Nice, thank you! 👍 🎉 

Let's :shipit:

**Pull request merged**